modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js

```diff
@@ -33,6 +32,7 @@ export default function DropdownMenu({
 	return (
 		<>
 			<ClayDropDownWithItems
+				className="dropdown-action"
```

Comment:
As I commented [here](https://github.com/liferay-frontend/liferay-portal/pull/211#discussion_r468448021) this component is not always an actions dropdown

Reply:
I added a conditional